<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>Pyenv on エンジニアもどきの技術メモ</title><link>https://chantsune.github.io/tags/pyenv/</link><description>Recent content in Pyenv on エンジニアもどきの技術メモ</description><generator>Hugo -- gohugo.io</generator><language>ja</language><lastBuildDate>Fri, 30 Oct 2020 08:35:24 +0000</lastBuildDate><atom:link href="https://chantsune.github.io/tags/pyenv/index.xml" rel="self" type="application/rss+xml"/><item><title>pyenvをアップデートしたらlibexec/pyenv: No such file or directoryとか言われた話</title><link>https://chantsune.github.io/articles/233/</link><pubDate>Tue, 02 Jul 2019 15:17:28 +0000</pubDate><guid>https://chantsune.github.io/articles/233/</guid><description>環境 macOS 10.14.3 pyenv 1.2.11 本題 ある時、気が向いて $ brew upgrade を実行したらpyenvのバージョンが上がった。 その直後、 $ python としたら、 Users/username/.pyenv/shims/python: line 21: /usr/local/Cellar/pyenv/1.2.11/libexec/pyenv: No such file or directory というエラー</description></item><item><title>MacのPythonで対話コンソールの履歴が遡れない</title><link>https://chantsune.github.io/articles/175/</link><pubDate>Sat, 23 Mar 2019 07:11:34 +0000</pubDate><guid>https://chantsune.github.io/articles/175/</guid><description>pyenvでpythonの環境を構築していたのですがある時から、対話コンソールで上下キーを押しても直前に打ったコマンドの履歴を遡れなくなって</description></item><item><title>ラズパイゼロにpyenvでpythonの環境構築する話</title><link>https://chantsune.github.io/articles/171/</link><pubDate>Mon, 25 Feb 2019 11:13:15 +0000</pubDate><guid>https://chantsune.github.io/articles/171/</guid><description>どこのご家庭にも一個や二個は余っている(？)ラズパイが自宅にもあったので、とりあえずPythonの環境を作るかってことでpyenvを使って環</description></item></channel></rss>